CHANGELOG.md

```diff
@@ -0,0 +1,134 @@
+- Breadcrumb UI component now has its arrow icons to the right of each breadcrumb-item. [#431] Breadcrumb arrows to the right
+- `<legend>` elements no longer have padding as default
+- The `.a-bordered-header` atom is now left-aligned rather than centered
+
+## Fixed
+
+- Selects, buttons, and inputs all have the same height, and share colors
+
+## Breaking
+
+- `.u-grid` classes no longer set `grid-gap`. Responsibility for `grid-gap` has been moved to `.u-gap`. You’ll need to add relevant grid-gap classes to your grid elements. Adding `.u-gap-m` to all your grid containers will result in an equivalent layout as before this change
+
+# [1.0.4] - 2021-03-17
+
+## Added
+
+- A filter component built using the existing dropdown and checkboxes
+- A filter component for applying free-text filters
+- An new icon page in our documentation, giving examples of all the icons used in our documentation
+
+
+## Changed
+
+- `.a-button--danger` no longer overrides base colors (only does so on hover/focus etc.), so you can use it in conjunction with other button types to make them “dangerous”
+- All icons in our documentation are now built with symbols from a centralised SVG file using `<use>`
+- All our button types now only alter the base button’s `border-color`, not the shorthand `border`, to ensure equal heights of buttons
+- `.a-button--nav` now responds to `aria-expanded`, so it can be used for dropdowns
+- `.a-button--tab` now responds to `aria-current`, so a tab-like interface can be presented when the tab switching involves navigating to a new URL
+- `.a-button--ui` now responds to `aria-current`
+- buttons now all use regular font-weight
+
+## Breaking
+
+- The `overflow-x` and `overflow-y` mixins no longer set the max width and height respectively. If you were making direct use of the mixins, you’ll need to add max widths or heights to your CSS, as needed.
+
+# [1.0.3] - 2021-03-15
+
+## Added
+
+- `.o-notification-center` component, that handles layout of notifications in the top right of the viewport, overlaying other content
+- `.a-flash` component to be used for displaying global updates at the top of the viewport, or at the top of a form, in the document flow
+- Negative margin utility classes `.u-margin-neg-#{$size}`
+
+## Changed
+
+- Most of our button types now only alter the base button’s `border-color`, not the shorthand `border`, to ensure equal heights of buttons
+- Input elements now only alter the base style’s `border-color`, not the shorthand `border`, to ensure equal heights and consistent appearance
+- Simplifies the way we import webfonts, and allows easier setup and overriding, using a more explicit `$bitstyles-webfont-variants` nested map. Filenames no longer need to match the previous format, though it’s advised
+
+# [1.0.2] - 2021-03-11
+
+## Added
+
+- `.u-border` utility classes
+- Storybok documentation now includes the `a11y-addon` for in-browser checking of some a11y low-hanging fruit
```

Comment:
I fixed the spelling of storybook, thanks :) But I left the word “fruit” without the “s” because here it’s the mass noun, not singular. Changing it to “low-hanging fruits” sounds odd (“fruits” plural is not common, it’s almost always referred to with the mass noun. Like I just used “it’s”, not “they’re”)